modules/apps/password-policies-admin/password-policies-admin-web/src/main/java/com/liferay/password/policies/admin/web/internal/display/context/EditPasswordPolicyAssignmentsManagementToolbarDisplayContext.java

```diff
@@ -111,6 +109,12 @@ public String getClearResultsURL() {
 		return clearResultsURL.toString();
 	}
 
+	public CreationMenu getCreationMenu() {
+		return CreationMenuBuilder.addPrimaryDropdownItem(
+			dropdownItem -> dropdownItem.putData("action", "addAssignee")
```

Comment:
Do we need to add `item.data.action="addAssignee"`? I don't think it is used anywhere. We do need to add _something_ just to have one primary dropdown item, does empty lambda work in Java? Something like 
```
return CreationMenuBuilder.addPrimaryDropdownItem(() -> {}):
```

Reply:
Going to try, I saw that we do need to add something as you said, but I'm wasn't sure the empty lambda would work

Reply:
I tried using 

```java
return CreationMenuBuilder.addPrimaryDropdownItem(() -> {}).build();
```

But the Java compiler says

```
/home/jc/Documents/portal/liferay-portal/modules/apps/password-policies-admin/password-policies-admin-web/src/main/java/com/liferay/password/policies/admin/web/internal/display/context/EditPasswordPolicyAssignmentsManagementToolbarDisplayContext.java:116: error: incompatible types: incompatible parameter types in lambda expression
```

So I changed it like [this](https://github.com/drewbrokke/liferay-portal/blob/c2ae813b8ed2cab963db8265552bab1428850271/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js#L167-L174), which compiles and works